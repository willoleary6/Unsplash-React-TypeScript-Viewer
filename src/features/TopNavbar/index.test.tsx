import React from "react";
import { TopNavbar } from ".";

import Enzyme, { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({ adapter: new Adapter() });

describe("TopNavbar render testing", () => {
    it("renders panel and takes snapshot to track jsx element changes, will also test for exceptions in element code", () => {
        const wrapper = shallow(<TopNavbar />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
