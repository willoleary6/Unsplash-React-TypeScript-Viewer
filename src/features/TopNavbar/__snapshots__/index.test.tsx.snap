// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNavbar render testing renders panel and takes snapshot to track jsx element changes, will also test for exceptions in element code 1`] = `
<nav
  className="navbar navbar-static-top navbar-fixed-top"
  role="navigation"
>
  <div
    className="navbar-header"
  >
    <SideNavbarToggle />
  </div>
</nav>
`;
